backend/app/assets/javascripts/spree/backend/views/quick_switch.js

```diff
@@ -0,0 +1,71 @@
+Spree.Views.QuickSwitch = Backbone.View.extend({
+  events: {
+    "shown.bs.modal": "onShow",
+    "hidden.bs.modal": "onHide",
+    "submit form": "onSubmit"
+  },
+
+  initialize: function() {
+    this.$el = $(this.el);
+    this.$dialog = $("[data-js='quick-switch-dialog']", this.$el);
+    this.$form = $("form", this.$el);
+    this.$input = $("[data-js='quick-switch-input']", this.$el);
+    this.$tooltip = $("[data-js='quick-switch-tooltip']", this.$el);
+		this.render();
+  },
+
+  triggerShortcut: function() {
+    this.onShortcut();
+  },
+
+  onShortcut: function() {
```

Comment:
I didn’t find any usage of this function besides from `triggerShortcut`. Could we combine these?

Reply:
I've found this to be a common pattern in JS-land. In this case something else triggers a shortcut and then our object knows what to do `onShortcut`. I'd prefer to keep it to keep the pattern alive but could be persuaded.

Reply:
If you would use a custom event name and listener, agreed. 

Anyways. It is great as it is right now. I will not block this great feature.